plugins/bulk_upload/drop_folder_xml/plugins/providers/ap_feed/batch/KApFeedDropFolderEngine.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * @package plugins.ApFeedDropFolder
+ */
+class KApFeedDropFolderEngine extends KFeedDropFolderEngine
+{
+	/**
+	 * The API Key for the AP feed being processed.
+	 * @var string
+	 */
+	protected $apiKey;
+	
+	const AP_FEED_PAGE_SIZE = 100;
+	
+	public function watchFolder(KalturaDropFolder $dropFolder)
+	{
+		/* @var $dropFolder KalturaApFeedDropFolder */
+		KalturaLog::info("Watching drop folder with ID [" . $dropFolder->id . "]");
```

Comment:
No need to use double quotes here since there is no processing , please use single.